features/cellular/framework/targets/UBLOX/AT/UBLOX_AT.cpp

```diff
@@ -116,6 +116,7 @@ nsapi_error_t UBLOX_AT::init()
         err = _at->at_cmd_discard("+CFUN", "=1"); // set full functionality
     }
 #endif
+    _at->unlock();
```

Comment:
Could be `return _at.unlock_return_error();` ?

Reply:
updated, thanks